### PR TITLE
chore: bump sdk version written in code

### PR DESCRIPTION
**Pull request by ewanharris**
## Description

I overlooked the version changes when reviewing #124, so bump those

## Review Checklist
- [x ] I have clicked on ["allow edits by maintainers"](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork).
- [ ] I have added documentation for new/changed functionality in this PR or in a PR to [openfga.dev](https://github.com/openfga/openfga.dev) [Provide a link to any relevant PRs in the references section above]
- [x] The correct base branch is being used, if not `main`
- [ ] I have added tests to validate that the change in functionality is working as expected

### CODE REVIEW

File: telemetry/telemetry.go [hunk 1]
```diff
@@ -49,7 +49,7 @@ var (
 func Configure(configuration *Configuration) (*Telemetry, error) {
 	return &Telemetry{
 		Metrics: &Metrics{
-			Meter:         otel.Meter("openfga-sdk/0.6.0"),
+			Meter:         otel.Meter("openfga-sdk/0.6.1"),
```
Comment (ewanharris) [inline telemetry/telemetry.go]: I just noticed this differs from `defaultUserAgent`, is taht expected?

Comment (evansims) [reply telemetry/telemetry.go]: It probably should be consistent; I'll update that

I've noticed our approach to the user agent strings differs across various SDKs, outside the scope of OpenTelemetry. We should do an alignment run across the SDKs at some point to make those consistent in all regards.

Comment (rhamzeh) [reply telemetry/telemetry.go]: The user agents are consistent across the SDKs, but the metrics name is unrelated and should not contain the user agent or the version.

`openfga-sdk` would work for that, and yes let's make the metric name across all the SDKs

Comment (evansims) [reply telemetry/telemetry.go]: @rhamzeh I believe we're currently using the `version` in the meter name across all SDKs. We can change it, but they'll all need updated.

Some of the SDKs have inconsistencies in some of their agent strings. Python, for example, reports the `User-Agent` as "openfga-sdk (python) X.Y.Z" in its OAuth2 class, although it is reported as "openfga-sdk python/X.Y.Z" in the API classes. I haven't checked all of them to see if this is the case everywhere.

**Review by evansims — APPROVED** [review]

**Pull request merged**